architecture/build-service.md

```diff
@@ -3,9 +3,9 @@
 
 ## Overview
 
-The Build Service is composed of controllers that create and configure build pipelines. The main input for the Build Service is a Component CR managed by the [Hybrid Application Service (HAS)](hybrid-application-service.md).
+The Build Service is composed of controllers that create and configure build pipelines. The main input for the Build Service is a Component CR managed by the Konflux UI.
```

Comment:
> managed by the Konflux UI.

Not necessarily. A user can create object via `kubectl`

Reply:
updated